src/app/beer_garden/router.py

```diff
@@ -426,6 +433,8 @@ def _pre_forward(operation: Operation) -> Operation:
         if wait_event:
             beer_garden.requests.request_map[operation.model.id] = wait_event
 
+        beer_garden.files.forward_file(operation)
```

Comment:
Can we move this to before we populate the request map (right above)? That way if something goes wrong with the file forward we won't have the wait_event hanging around in the map.